doc/nrf/protocols/esb/index.rst

```diff
@@ -302,3 +302,26 @@ Examples
 The |NCS| provides the following example application that shows how to use the ESB protocol:
 
 * :ref:`esb_prx_ptx`
+
+.. _esb_fast_ramp_up:
+
+Fast ramp-up
+============
+
+The ESB driver is designed for bidirectional data transfer between devices.
+This implies that the radio peripheral is continuously switching between reception and transmission mode.
+When switching to transmission or reception mode, the radio peripheral needs some time to start internal analog components.
+On nRF51 and nRF24L series, a hardware-limited 130 µs delay is implemented.
+If ESB connection is established only between nRF52 and/or nRF53 devices, this delay can be reduced to 40 µs.
+
+When ``use_fast_ramp_up`` parameter's value is true, the fast ramp-up will be enabled, resulting in reducing ramp-up delay to 40 µs.
+
+.. _esb_never_disable_tx:
+
+Experimental feature: Never disable transmission stage
+======================================================
+
+If reducing delay between TX FIFO filling and reception to minimum is needed, enabling fast ramp-up may not be sufficient.
+In this case, you can use the :kconfig:option:`CONFIG_ESB_NEVER_DISABLE_TX` Kconfig option.
+This option changes the ESB driver's behavior so that if a packet is not acknowledged, the radio peripheral will remain in TXIDLE state instead of TXDISABLE when transmission is pending.
```

Comment:
```suggestion
It changes the ESB driver's behavior.
If a packet is not acknowledged, the radio peripheral remains in TXIDLE state instead of TXDISABLE when transmission is pending.
```